src/content/getting-started/shared-installation/index.md

```diff
@@ -40,9 +40,9 @@ For normal Giant Swarm installations, the customer can decide which identity pro
 
 In a shared installation, all customers use GitHub as an identity provider and configure a team in a Github organization to include all the users who need access to the Giant Swarm Management API and user interfaces. Find instructions regarding the setup of your organization and team below.
 
-### Monitoring
+### Observability
 
-A shared installation is monitored and managed by Giant Swarm staff, just like any other installation. However, we currently do not provide access to our [Monitoring]({{< relref "/ui-api/monitoring" >}}) for users of a shared installation.
+A shared installation is monitored and managed by Giant Swarm staff, just like any other installation. However, we currently do not provide access to our [Observability]({{< relref "/ui-api/observability" >}}) for users of a shared installation.
```

Comment:
Even though it is almost the same, "access to our observability" sounds strange to me.

Maybe something like "access to our standard observability features"?